apps/explorer/lib/explorer/chain/internal_transaction/type.ex

```diff
@@ -11,8 +11,19 @@ defmodule Explorer.Chain.InternalTransaction.Type do
    * `:reward`
    * `:selfdestruct`
    * `:stop`
+   #{if Application.compile_env(:explorer, :chain_type) == "arbitrum" do
+    """
+      * `:arbitrum`
```

Comment:
Looks like it should be
```suggestion
      * `:invalid`
```
Also the `create2` is missing in this typedoc.